docker/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+pid_file='/var/run/gunicorn/gunicorn.pid'
+
+function stop_container {
+    service nginx stop
+    gunicorn_pid=$(cat ${pid_file})
+    kill -TERM $(cat ${pid_file})
+}
+
+trap stop_container SIGTERM
+
+echo 'Installed handler' 1>&2
+
+service nginx start
+
+su -c ". /home/mahiru/.env/bin/activate && gunicorn --pid ${pid_file} --access-logfile /var/log/gunicorn/access.log --error-logfile /var/log/gunicorn/error.log --capture-output --bind 127.0.0.1:8000 'proof_of_concept.rest.registry:wsgi_app()'" mahiru &
```

Comment:
All the arguments here could be arguments for this script, which you could set in the docker file.

Reply:
Ah, good point. Most of them are fine as-is, but I had noticed that the app function entry point is the only difference between the registry image and the site image. So moving that to the Docker file would save some duplication. Will fix.